Modules/Devices/test/Devices.Application.Tests/Tests/Devices/Commands/DeleteDevice/HandlerTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Devices.Application.Devices.Commands.DeleteDevice;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Entities;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Devices.Commands.DeleteDevice;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Deletes_device_owned_by_identity()
+    {
+        // Arrange
+        var startTime = DateTime.UtcNow;
+
+        var identity = TestDataGenerator.CreateIdentity();
+        var device = new Device(identity);
+
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        A.CallTo(() => mockIdentitiesRepository.GetDeviceById(device.Id, A<CancellationToken>._, A<bool>._)).Returns(device);
+        A.CallTo(() => mockIdentitiesRepository.Update(device, A<CancellationToken>._)).DoesNothing();
```

Comment:
Why this line?

Reply:
I imagined I need to fake it in order to make the test work.